srcpkgs/python-PyQt5-webengine/template

```diff
@@ -63,29 +63,20 @@ EOF
 		make ${makejobs}
 	done
 }
+
 do_install() {
-	# install python-pyqt5
 	make -C pyqt5-${py2_ver} DESTDIR=${DESTDIR} INSTALL_ROOT=${DESTDIR} install
-
-	# install python3-pyqt5
-	make -C pyqt5-${py3_ver} DESTDIR=${DESTDIR} INSTALL_ROOT=${DESTDIR} install
-}
-python-PyQt5-webengine_package() {
-	lib32disabled=yes
-	depends="${sourcepkg}-${version}_${revision}
-	 python-PyQt5-webchannel-${version}_${revision}"
-	pkg_install() {
-		vmove ${py2_sitelib}/PyQt5/QtWebEngine.so
-		vmove ${py2_sitelib}/PyQt5/QtWebEngineCore.so
-		vmove ${py2_sitelib}/PyQt5/QtWebEngineWidgets.so
-	}
+	rm -rf ${DESTDIR}/usr/share
 }
+
 python3-PyQt5-webengine_package() {
 	lib32disabled=yes
 	depends="python3-PyQt5-${version}_${revision}
 	 python3-PyQt5-webchannel-${version}_${revision}"
 	short_desc="${short_desc/Python2/Python3}"
 	pkg_install() {
+		make -C pyqt5-${py3_ver} DESTDIR=${DESTDIR} INSTALL_ROOT=${DESTDIR} install
```

Comment:
This should stay in `do_install`, `pkg_install` should only move files from `DESTDIR` to `PKGDESTDIR`.

Reply:
@Duncaen Ok, I've made those changes.  Thanks for the feedback.